content/verification/address-validation/release-notes/2024-05-15.md

```diff
@@ -0,0 +1,11 @@
+We released version 3 (v3) of address validation. Updates include:
+- `/validate` and `/candidate` endpoints
+
+     - Removing support for US Congressional District. 
```

Comment:
@dsteeve are these two (Removing support...) and (Adding country code..) supposed to be sub-bullets of '/validate' and '/candidate'? Just want to double check.

Reply:
They are sub-bullets to these endpoints.  If you look at address-validation v1, there was a congressional district param, and that is no longer there, and country code handling is now different as well.

Reply:
Awesome, I'll resolve!